service/src/test/java/bio/terra/appmanager/api/AdminControllerTest.java

```diff
@@ -77,6 +83,48 @@ void testCreate_204() throws Exception {
         capture_chartVersions.getValue().get(0), chartName, chartVersion, null, null, null);
   }
 
+  @Test
+  void testInvalidChartVersionPost() throws Exception {
```

Comment:
```suggestion
  void testCreate_invalidChartVersion() throws Exception {
```
wondering your thoughts on keeping the naming consistent